Monika After Story/game/definitions.rpy

```diff
@@ -7338,8 +7350,20 @@ default persistent.chess_strength = 3
 default persistent.closed_self = False
 default persistent._mas_game_crashed = False
 default persistent.seen_monika_in_room = False
-default persistent.ever_won = {'pong':False,'chess':False,'hangman':False,'piano':False}
-default persistent.sessions={'last_session_end':None,'current_session_start':None,'total_playtime':datetime.timedelta(seconds=0),'total_sessions':0,'first_session':datetime.datetime.now()}
+default persistent.ever_won = {
+    "pong": False,
+    "chess": False,
+    "hangman": False,
+    "piano": False,
+    "nou": False
+}
```

Comment:
```suggestion
#TODO: Add a couple functions to register new items to this and check safely. Assuming the key not being present is a False ever won.
default persistent.ever_won = {
    "pong": False,
    "chess": False,
    "hangman": False,
    "piano": False,
    "nou": False
}
```

Reply:
I don't think we can do this, we'd have to modify the `default` statement.

Reply:
Nah, we can. Not too difficult. Ideally we can just default this to an empty dict and add things as needed. Especially since a missing key results in `False`.